contracts/src/CrossChainApplications/NativeTokenBridge/ERC20TokenSource.sol

```diff
@@ -166,6 +117,42 @@ contract ERC20TokenSource is
         });
     }
 
+    /**
+     * @dev See {ITeleporterReceiver-receiveTeleporterMessage}.
```

Comment:
think it might be worthwhile to mention `TeleporterUpgradeable.receiveTeleporterMessage` as well, or just replace it.